KerbalStuff/blueprints/profile.py

```diff
@@ -20,7 +20,8 @@ def view_profile(username):
             if not current_user.admin:
                 abort(401)
     mods_created = sorted(user.mods, key=lambda mod: mod.created, reverse=True)
-    if not current_user or current_user.id != user.id:
+    # Remove unpublished mods from the list if it's not the accessing's user's own profile, or it is and admin.
+    if not current_user or (current_user.id != user.id and not current_user.admin):
```

Comment:
The condition for the if block should be `if (not current_user or current_user.id != user.id) and not current_user.admin`, otherwise unpublished mods wouldn't show up for yourself even if you're not an admin.

Reply:
No I think it's correct.
We start with the full list.
The unpublished mods are _removed_ from the list if:
1) I'm not logged in *or*
2) I'm not the author *and* I'm not an admin.
	  That means if I _am_ the author, the list is not reduced,
	  if I _am_ and admin, the list is not reduced.